site/en/docs/puppeteer/faq/index.md

```diff
@@ -74,7 +74,9 @@ npm install puppeteer-core@chrome-71
 
 Find the version using one of the following ways:
 
-- Look for the `chromium` entry in [revisions.ts](https://github.com/puppeteer/puppeteer/blob/main/src/revisions.ts). To find the corresponding Chromium commit and version number, search for the revision prefixed by an `r` in [OmahaProxy](https://omahaproxy.appspot.com/)'s "Find Releases" section.
+- Look for the `chromium` entry in [revisions.ts](https://github.com/puppeteer/puppeteer/blob/main/src/revisions.ts). To find the corresponding Chromium commit and version number, append the entry onto 
```

Comment:
This link returns 404. I don't even know what this is, so I didn't have much luck finding a new link.